e2e/FR16.e2e-spec.ts

```diff
@@ -16,12 +16,15 @@ describe('Zonemaster test FR16 - [The advanced view should have a text describin
     expect(element(by.css('.alert.alert-info')).element(by.css('a')).getAttribute('fragment')).toBe('q12');
   });
 
-  it('should have a description text in multi languages', () => {
-    utils.setLang('en');
+  it('should have a description text in multi languages', async () => {
+    await utils.setLang('en');
+    await browser.sleep(100);
```

Comment:
Is that seconds? Isn't 100 s beyond what is needed?

Reply:
It is in milliseconds

Reply:
> It is in milliseconds

Then it is more than fine. :-)